downstream/modules/platform/proc-operator-link-components.adoc

```diff
@@ -1,3 +1,35 @@
 [id="operator-link-components_{context}"]
 
-= Linking your components to AnsibleAutomationPlatform  
\ No newline at end of file
+= Linking your components to the central configuration  
+
+Once you have the {PlatformNameShort} operator installed in your namespace you can set up your *AnsibleAutomationPlatform* instance.
+Then link all the platform components to a single user interface. 
+
+.Procedure 
+. Go to your {PlatformNameShort} operator and click btn:[Details]. 
+. On the *AnsibleAutomationPlatform* tile click btn:[Create instance].
+. From the *Create AnsibleAutomationPlatform* page enter a name for your instance in the *Name* field.
+. Click btn:[YAML view] and paste the following:
++
+----
+spec:
+  controller:
+    disabled: false
+
+  eda:
+    disabled: false
+
+  hub:
+    disabled: false
+    storage_type: file
+    file_storage_storage_class: nfs-local-rwx
+    file_storage_size: 10Gi
+
+----
+. Click btn:[Create].
+
+.Verification
+To verify if all instances deployed correctly go to your Ansible Automation Platform operator deployment and click btn:[All instances].
+Here you will see the *AnsibleAutomationPlatform* instance and the *AutomationController*, *EDA*, and *AutomationHub* instances deployed.
+
+Alternatively you can check via the command line, run: `oc get route` 
```

Comment:
Flagged in Vale for "via" - Depending on the context, consider using 'through', 'by', 'from', 'on', or 'by using' rather than 'via'